.eslintrc.json

```diff
@@ -3,7 +3,8 @@
     "browser": true,
     "es6": true,
     "node": true,
-    "jest": true
+    "jest": true,
+    "mongo": true
```

Comment:
what does adding mongo do ?

Reply:
It keeps ESLint from throwing 'not found' errors on mongo functions used in the global scope.